tools/import/import.js

```diff
@@ -200,3 +201,5 @@ function addJobsList(jobs) {
     }
   });
 })();
+
+sampleRUM.enhance();
```

Comment:
This is wrong and will load the enhancer twice... The enhance method is already called in `loadEager` because `script.js` is loaded in `index.html`. @amol-anand you can revert the commit / merge.